utils/CacheUtils.tsx

```diff
@@ -72,6 +72,10 @@ let cacheUtils: CacheUtils = {
 }
 export default cacheUtils
 
+export function getCacheContolHeader() {
+    return 'public, max-age=300, s-maxage=60, stale-while-revalidate=59'
```

Comment:
```suggestion
    return 'public, max-age=60, s-maxage=20, stale-while-revalidate=19'
```
we update data more frequently